src/main/java/oncall/OncallController.java

```diff
@@ -0,0 +1,42 @@
+package oncall;
+
+import java.util.List;
+import oncall.domain.crew.Crews;
+import oncall.domain.crew.Schedule;
+import oncall.domain.crew.ScheduleMaker;
+import oncall.domain.day.WorkDays;
+import oncall.exception.handler.RetryHandler;
+import oncall.view.InputView;
+import oncall.view.OutputView;
+
+public class OncallController {
+    public void run() {
+        WorkDays workDays = RetryHandler.getOrRetry(() -> getCalendar());
+        ScheduleMaker scheduleMaker = RetryHandler.getOrRetry(() -> getCrews());
```

Comment:
메서드 참조를 사용하면 더 간략하게 작성할 수 있습니다.
```suggestion
        WorkDays workDays = RetryHandler.getOrRetry(this::getCalendar());
        ScheduleMaker scheduleMaker = RetryHandler.getOrRetry(this::getCrews());
```

Reply:
사실 이걸 메서드 참조로 바꿀까 고민을 했었는데요..
`Controller`를 하나의 인스턴스로 판단해 `this`를 사용하는 것이 아무래도 조금은 어색해서...? 그냥 람다식으로 두게 되었습니다.

분명 스프링에서도 `Controller`를 포함한 스프링 빈들은 모두 각각의 싱글톤 객체로 관리되기에 `this`를 사용하는 것이 자연스러울 법 한데, 이상하게 제 마음에선 뭔가 어색하게 느껴지더라고요...ㅎㅎ

Reply:
오..? `Controller`를 하나의 주체적인 객체로 보지 않는 것은 흥미롭네요. Java에서 `new`를 사용해 생성된 객체들은 모두 주체적이라고 생각했어요 (그래서 생성자를 감추고 정적 팩토리 메서드를 사용하는 경우에는 주체적인 메서드를 작성하지 않으려 합니다), 여러 측면에서 생각할 수 있어서 재밌습니다 😃

Reply:
아, 어색하게 느껴지는 이유가 하나 더 있는 것 같아요!
다른 객체들은 모두 하나의 작은 책임 영역을 가지고 있는 도메인 객체들인데요! (날짜와 관련된 책임 / 근무자를 표현하는 책임 등등)
그와는 반대로 `Controller`는 어떻게 보면 "흐름을 제어" 하는 역할을 하는 객체이기에 다른 도메인 객체와 받아들이는 느낌이 달랐던 것 같네요!